src/main/js/controls/Line/Line.js

```diff
@@ -158,23 +169,33 @@ class Line extends GraphContent {
      * @inheritdoc
      */
     resize(graph) {
-        if (utils.notEmpty(this.dataTarget.regions)) {
+        if (
+            utils.notEmpty(this.dataTarget.regions) ||
+            !utils.isEmptyArray(this.dataTarget.valueRegionSubset)
+        ) {
             if (graph.content.length > 1 && !graph.config.shouldHideAllRegion) {
                 if (areRegionsIdentical(graph.svg)) {
                     graph.config.shouldHideAllRegion = false;
                 } else {
-                    hideAllRegions(graph.svg);
                     graph.config.shouldHideAllRegion = true;
                 }
             }
         } else {
-            hideAllRegions(graph.svg);
             graph.config.shouldHideAllRegion = true;
         }
+        if (graph.config.shouldHideAllRegion) {
+            hideAllRegions(graph.svg);
+        }
```

Comment:
May i know what is the purpose of using separate if statement? Can we just call hideAllRegions method directly in else ?

Reply:
While developing I found an issue.
If there are more than two lines and all has regions in it. Ideally the graph should hide all regions. Instead here it was hiding only first two regions. And other regions were being displayed.
So I gotta move it out of that if condition to fix this issue.